scripts/fixUseClient.js

```diff
@@ -1,4 +1,12 @@
 const fs = require('fs');
+const glob = require('glob');
+
+let iconFiles = glob.sync('packages/@react-spectrum/s2/{icons,illustrations}/**/*.{cjs,mjs}');
+for (let f of iconFiles) {
+  let contents = fs.readFileSync(f, 'utf8');
+  contents = contents.replace(/['"]use client['"];?/g, '');
+  fs.writeFileSync(f, contents);
+}
```

Comment:
wait so is it a problem if we remove the `use client` from the icons? https://github.com/adobe/react-spectrum/pull/9126#discussion_r2483014933 seems to indicate that they should be client components but I'm not quite familiar with the nature of the nextJS build issue

Reply:
yeah, so it's supported in our docs, which build from src. but it's not supported outside of that yet, so we just remove it, that's why it's removed from the other two files in this script as well, external users must import s2 components in a file that already has use client

don't worry, I asked the same questions when chatting with Devon about how to fix this